langfuse/callback.py

```diff
@@ -466,6 +466,8 @@ def __on_llm_action(
                 )
             if kwargs["invocation_params"]["_type"] in ["anthropic-llm", "anthropic-chat"]:
                 model_name = "anthropic"  # unfortunately no model info by anthropic provided.
+            elif kwargs["invocation_params"]["_type"] == "amazon_bedrock":
+                model_name = "claude"  # unfortunately no model info by Bedrock provided.
```

Comment:
@kobrinartem thanks for the contribution! Did you test whether Langchain somehow provides the model name?

Reply:
I didn't find the capability. How does it work for OpenAI?

Reply:
@maxdeichmann can you guide me how to improve it? Is it good enough as is?

Reply:
Sure. Can you add a simple example of using bedrock to `test_langchain.py`?
I would want to check whether we can somehow find the model in the kwargs or somewhere else. It would be great to capture the model_id here: https://python.langchain.com/docs/integrations/llms/bedrock